src/test/resources/pages/newlyJoinedInstructorFeedbackSessionSuccessEdited.html

```diff
@@ -213,7 +213,7 @@ <h5 class="col-sm-2 margin-top-0">
                 <div class="col-sm-10">
                   <div class="panel panel-default panel-body mce-content-body content-editor" id="instructions" spellcheck="false">
                     <p>
-                      updated instructions
+                      Please give your feedback based on the following questions.
```

Comment:
No, html is not used now. In fact we can now remove this file as it is not used.